### ISSUE TITLE
Add a set of tutorials for MarkBind

### ISSUE THREAD

**Issue by lhw-1** [MarkBind/markbind#2257]
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### Any related issues?

#2214

### What is the area that this feature belongs to?

Author Usability, Documentation

### Is your feature request related to a problem? Please describe.

As mentioned in #2214, a set of tutorials for users new to MarkBind would be great to have. 

For some context (just to be clearer), there is a difference between a quick start, a feature overview, and a tutorial, even though they may seem to have overlaps at times for static site generators. To illustrate with examples, [Gatsby](https://www.gatsbyjs.com/docs/) has separate pages / sets of pages for their [quick start](https://www.gatsbyjs.com/docs/quick-start/), which shows the installation process; their [feature overview](https://www.gatsbyjs.com/docs/how-to/), which is a simple list of features available; and their [tutorial](https://www.gatsbyjs.com/docs/tutorial/), which is a lot more detailed and step-by-step as compared to their quick start. Hugo includes the [quick start](https://gohugo.io/getting-started/quick-start/) as part of their ["tutorial"](https://gohugo.io/getting-started/) pages. Jekyll also separates [quick start](https://jekyllrb.com/docs/) and [tutorial](https://jekyllrb.com/docs/step-by-step/01-setup/). So there is definitely precendent for separating these sections explicitly.

In comparison, for MarkBind, we have the [quick start](https://markbind.org/userGuide/gettingStarted.html) and the [feature overview](https://markbind.org/userGuide/authoringContents.html), but not a tutorial.

### Describe the solution you'd like

We can mirror the existing [Dev Guide - Onboarding Bootcamp](https://markbind.org/devdocs/devGuide/bootcamp/intro.html) in terms of structure.

**Comment by jovyntls** [comment]
Relevant: #835

Are these similar?

**Comment by lhw-1** [comment]
> Relevant: #835
> 
> Are these issues similar? In terms of the difference between a quick start, a feature overview, and a tutorial, both issues seem to point towards a tutorial

Definitely! It was brought up in #2214 (and I forgot to initially include it in relevant issues - now updated).

The two issues definitely sound very similar, but the main difference seems to be that tutorials target a step-by-step process on how to install / set-up, write proper syntax, include components, etc. while recipes target potential use cases to achieve a specific goal (which assumes that users are already familiar with the syntax). 

To illustrate the difference, an example is Gatsby - as brought up in #835, they have a section for recipes (["How-to" section](https://www.gatsbyjs.com/docs/how-to/)), but this is preceded by their [Tutorials section](https://www.gatsbyjs.com/docs/tutorial/). 

**Description of their Tutorials section:** The goal of this tutorial is to help you create a mental model for how Gatsby sites work by building and deploying a blog site using MDX. Along the way, you'll learn how to use Gatsby plugins, the GraphQL data layer, and more!

**Description of their How-to section:** Practical step-by-step guides to help you achieve a specific goal. Most useful when you're trying to get something done.

Of course, if we think that we are over-bloating the documentation by having four sections cover possibly overlapping content (quick start, feature overview, tutorial, and recipes), we can always decide not to have a tutorial section.

**Comment by kaixin-hc** [comment]
I think if each section can be read somewhat independently, and we clearly signal what is in each, having multiple ways to get started is a good thing! We can even use MarkBind features to improve the experience for each (eg - using quizzes in the tutorial)
